.devcontainer/libraries/devcontainer.json

```diff
@@ -0,0 +1,47 @@
+// For format details, see https://aka.ms/devcontainer.json.
+{
+	"name": "C# (.NET)",
+	"build": {
+		"dockerfile": "Dockerfile",
+		"args": {
+			// Update 'VARIANT' to pick a .NET Core version: 3.1, 6.0
+			// Append -bullseye or -focal to pin to an OS version.
+			"VARIANT": "6.0-focal"
```

Comment:
Any reason that `6.0-jammy` (Ubuntu 22.04) can't be used instead of `6.0-focal` (Ubuntu 20.04)?

Reply:
For the `dotnet` image, `jammy` is not supported yet. You can see the complete list of available tags [here](https://mcr.microsoft.com/v2/devcontainers/dotnet/tags/list)